wiki/Continuous_integration.md

```diff
@@ -31,8 +31,8 @@ Currently you have 3 options
 1.  Visit our [buildbot web site](http://testing.open-bio.org/biopython)
 2.  Subscribe to the integration [RSS
     feed](http://testing.open-bio.org/biopython/rss)
-3.  Subscribe to the biopython-dev mailing list where important errors
-    are automatically forward (under construction)
+3.  Subscribe to the [Biopython mailing list](mailto:biopython@biopython.org)
+    where important errors are automatically forward (under construction)
```

Comment:
This whole section needs to be updated, but I'll do that after your changes...

Reply:
See 6a9ddfd9a19ad8cffff7ff515481e72e1c0dc10f and https://biopython.org/wiki/Continuous_integration